airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/staging/SerialFlush.kt

```diff
@@ -79,11 +79,11 @@ object SerialFlush {
         return FlushBufferFunction {
             pair: AirbyteStreamNameNamespacePair,
             writer: SerializableBuffer ->
-            log.info(
-                "Flushing buffer for stream {} ({}) to staging",
-                pair.name,
-                FileUtils.byteCountToDisplaySize(writer.byteCount)
-            )
+            log.info {
```

Comment:
```suggestion
            log.info {"Flushing buffer for stream ${pair.name} (${FileUtils.byteCountToDisplaySize(writer.byteCount)}) to staging"}
```

not actually sure I understand what's written right now - why is there `"${"..."}"`? (similar question for the other log.error call)

Reply:
https://kotlinlang.org/docs/strings.html#string-templates